### ISSUE TITLE
Errors and exceptions page should not be translatable

### ISSUE THREAD

**Issue by GreenLunar** [searxng/searxng#791]
Errors and exceptions page `/stats?engine=pixelfed` should not be translatable, nor subjected to RTL layout (i.e. force LTR).

The report is expected to be in English, yes?

**Comment by return42** [comment]
> The report is expected to be in English, yes?

Yes! .. I'm LTR reader, do you see RTL issues in the `/stats` pages?
